### DISCUSSION TITLE
Enabling console.log() in the theme files

### DISCUSSION THREAD

**Discussion by yoscbd** [holger1411/understrap#1829] [Q&A] [answered]
Hi,
I know we cant sipmly use console.log() in react - it wont show up unlike alert().

when working with 
understrap dev env what is the best practice to have console.log() working?

**Comment by IanDelMar** [comment, marked as answer]
```diff
- "drop_console": true,
+ "drop_console": false,
```
in your terser.config.json

**Reply by yoscbd** [reply]
Thank you!!!! did the trick :)
